SpatialGDK/Documentation/content/toolbars.md

```diff
@@ -112,9 +92,11 @@ The toolbar settings are in **Edit** > **Project Settings** > **SpatialOS GDK fo
 
 | Setting | Description |
 | --- | --- |
-| Generate default launch config | If checked, the GDK creates a [launch configuration file]({{urlRoot}}/content/glossary#launch-configuration-file) by default when you launch a local deployment through the toolbar. |
+| Command line flags for local launch | Command line flags passed in to `spatial local launch`. |
 | Launch configuration | The [launch configuration file]({{urlRoot}}/content/glossary#launch-configuration-file) to use when running `spatial local launch` using the **Start** button. |
 | Stop on exit |  If enabled, shuts down running deployments when you close the Unreal Editor. |
+| Generate default launch config | If checked, the GDK creates a [launch configuration file]({{urlRoot}}/content/glossary#launch-configuration-file) by default when you launch a local deployment through the toolbar. |
```

Comment:
Is there any reason why this says "If checked", whereas the row above says "If enabled"?

Reply:
(If there's no reason for them to be different, I'd suggest choosing one and making them all consistent. If there is a reason, you can ignore me!)

Reply:
Sure will updated now:)